apps/test/unit/templates/rubrics/LearningGoalsTest.jsx

```diff
@@ -65,6 +69,136 @@ describe('LearningGoals', () => {
+};
+
+describe('LearningGoals - React Testing Library', () => {
+  it('renders EvidenceLevels without canProvideFeedback', () => {
+    render(<LearningGoals learningGoals={learningGoals} teacherHasEnabledAi />);
+
+    // This text only shows up within EvidenceLevels when canProvideFeedback is false
+    expect(screen.getByText('Rubric Scores')).to.exist;
+
+    // First learning goal is visible
+    expect(screen.getByText('Learning Goal 1')).to.exist;
+    expect(screen.getByText(/lg one none/)).to.exist;
+    expect(screen.getByText(/lg one limited/)).to.exist;
+    expect(screen.getByText(/lg one convincing/)).to.exist;
+    expect(screen.getByText(/lg one extensive/)).to.exist;
+  });
+
+  describe('when aiConfidenceExactMatch is high', () => {
+    const myAiEvaluations = [
+      {
+        ...aiEvaluations[0],
+        aiConfidenceExactMatch: 3,
+      },
+    ];
+    it('highlights one bubble', () => {
+      render(
+        <LearningGoals
+          learningGoals={learningGoals}
+          aiEvaluations={myAiEvaluations}
+          studentLevelInfo={studentLevelInfo}
+          teacherHasEnabledAi
+          canProvideFeedback
+        />
+      );
+      expect(getSuggestedButtonNames()).to.deep.equal(['Convincing']);
+    });
+    it('shows only one evaluation level in written summary', () => {
+      render(
+        <LearningGoals
+          learningGoals={learningGoals}
+          aiEvaluations={myAiEvaluations}
+          studentLevelInfo={studentLevelInfo}
+          teacherHasEnabledAi
+          canProvideFeedback
+        />
+      );
+      screen.getByText(
+        'Stella has achieved Convincing Evidence for this learning goal.'
+      );
+    });
+    it('shows exact-match confidence level', () => {
+      render(
+        <LearningGoals
+          learningGoals={learningGoals}
+          aiEvaluations={myAiEvaluations}
+          studentLevelInfo={studentLevelInfo}
+          teacherHasEnabledAi
+          canProvideFeedback
+        />
+      );
+      screen.getByText('AI Confidence: high');
+    });
+  });
+
+  describe('when aiConfidenceExactMatch is low', () => {
+    it('highlights two bubbles', () => {
+      render(
+        <LearningGoals
+          learningGoals={learningGoals}
+          aiEvaluations={aiEvaluations}
+          studentLevelInfo={studentLevelInfo}
+          teacherHasEnabledAi
+          canProvideFeedback
+        />
+      );
+      expect(getSuggestedButtonNames().sort()).to.deep.equal(
+        ['Convincing', 'Extensive'].sort()
+      );
+    });
+    it('shows two evaluation levels in written summary', () => {
+      render(
+        <LearningGoals
+          learningGoals={learningGoals}
+          aiEvaluations={aiEvaluations}
+          studentLevelInfo={studentLevelInfo}
+          teacherHasEnabledAi
+          canProvideFeedback
+        />
+      );
+      screen.getByText(
+        'Stella has achieved Extensive or Convincing Evidence for this learning goal.'
+      );
```

Comment:
Do you need an expect here?

Reply:
I believe the answer is no! admittedly I had to look this up to know the answer... but it looks like RTL "queries" starting with "get" will throw if the element is not found: https://testing-library.com/docs/queries/about/

Reply:
I was able to confirm that all 3 of the tests using "get" queries do fail if I ask them to look for a nonexistent string.

Reply:
OK, good to know!